docs/source/about/changelog.rst

```diff
@@ -4,6 +4,15 @@ Changelog
 Current development version
 ---------------------------
 
+v2.0.0 (upcoming release)
+-------------------------
+
+- **BREAKING** Change the :code:`GraphQLDateTime` scalar type from being timezone-aware aware to
```

Comment:
nit: `timezone-aware aware`

What's the difference between this changelog and the other `CHANGELOG.rst`? Are they always kept in sync with one another?

Reply:
Thanks for pointing the typo.

They have the same content and they should be kept in sync with each other. `changelog.rst` is meant to be the changelog for readthedocs.  There is still no link in the `README.rst` pointing to the readtheddocs, so the readthedocs documentation is not really discoverable. 
That's why we still have an identical `CHANGELOG.rst` in the top level directory. 

So unfortunately until we fully release readthedocs,  most of the documentation is going to be duplicated in to places.

This is what the readthedocs documentation looks like right now:
https://graphql-compiler.readthedocs.io/en/latest/

Right now we are only missing the documentation that would go in the `README.rst` once we move to readthedocs.